src/performance_profiles.jl

```diff
@@ -26,6 +26,17 @@ function performance_ratios(T :: Array{Float64,2}; logscale :: Bool=true)
   for p = 1 : np
     r[p, :] = T[p, :] / minperf[p];
   end
+  # Use a draw tolerance of `drawtol` (in percentage).
+  if (drawtol > 0. && drawtol < 1.)
```

Comment:
Why the limit `< 1`?
Also
```suggestion
  if (drawtol > 0 && drawtol < 1)
```